### PR TITLE
Docs updated

### PR DESCRIPTION
**Pull request by AleksaMCode**
Some additional details added to the documentation.

### CODE REVIEW

File: content/setup/_index.md [hunk 1]
```diff
@@ -143,6 +142,7 @@ server {
   ssl_stapling on;
   ssl_stapling_verify on;
 
+	# location of the build of file-picker
```
Comment (avivace) [inline content/setup/_index.md]: I think indentation here should be fixed to use space like the rest of the document

File: content/setup/_index.md [hunk 2]
```diff
@@ -119,10 +115,13 @@ Then, add the following to the `http` section of the `/etc/nginx/nginx.conf`.
...
+[...]
+
 server {
-  listen 80 default_server;
-  listen [::]:80 default_server;
-  server_name YOURHOSTNAME;
+  listen 8080 default_server;
+  listen [::]:8080 default_server;
+    # hostname we will be serving on, e.g. localhost
+    server_name YOURHOSTNAME;
```
Comment (avivace) [inline content/setup/_index.md]: indent here is 4 spaces, correct to 2
```suggestion
  server_name YOURHOSTNAME;
```

File: content/setup/_index.md [hunk 3]
```diff
@@ -119,10 +115,13 @@ Then, add the following to the `http` section of the `/etc/nginx/nginx.conf`.
...
 ```conf
+[...]
+
 server {
-  listen 80 default_server;
-  listen [::]:80 default_server;
-  server_name YOURHOSTNAME;
+  listen 8080 default_server;
+  listen [::]:8080 default_server;
+    # hostname we will be serving on, e.g. localhost
```
Comment (avivace) [inline content/setup/_index.md]:
```suggestion
  # hostname we will be serving on, e.g. localhost
```

File: content/setup/_index.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ To set up your own File-picker instance, you need to do the following:
 - Edit the configuration files
 - Set up a web server
 
-This guide has been tested in CentOS 7 and Ubuntu 22.04, but it should work in
+This guide has been tested in CentOS 7, Ubuntu 22.04 and Linux Mint 21.1, but it should work in
```
Comment (avivace) [inline content/setup/_index.md]: Linux Mint is based on Ubuntu, I don't think they are different enough to deserve to be distinguished here

**Review by avivace — CHANGES_REQUESTED** [review]
Thanks @AleksaMCode !

Before flagging this as ready (and ask @javfg to give a review) branch should be rebased.

**Comment by javfg** [comment]
Thanks for the contribution! :tada:

I think @avivace review covers most of it, I just added a small detail. Should be ready to merge as soon as those are fixed.

**Comment by avivace** [comment]
@javfg Thanks for taking a look! We just noticed that the Hugo build is not in the pipeline (so after merging nothing will change on https://filepicker.cernbox.cern.ch/docs, I guess). Should we commit changes to `public` too? Or you want to do that once we merge the changes on the src?

**Comment by javfg** [comment]
Ah yup, please add the new build! I'll set up a proper pipeline when I have some free time. :)

**Comment by javfg** [comment]
No need to review these changes, as it is an update of the theme used in the docs.

**Pull request merged**